packages/ui/src/components/sha-badge.tsx

```diff
@@ -43,10 +43,10 @@ function Icon({ ...props }: IconProps) {
 
   return (
     <div
-      onClick={handleClick && handleClick}
       role="button"
       tabIndex={0}
```

Comment:
Instead of attaching a click handler to a `div` and try to get it to pretend to be a `button`, can we just use a `button`?